plugins/cpp/model/include/model/cppastnode.h

```diff
@@ -57,6 +57,7 @@ struct CppAstNode
     LocalTypeLoc,
     TypedefTypeLoc,
     InheritanceTypeLoc,
+    Resolution,
```

Comment:
These nodes are created from Shadow Decls. So why are they called "resolution"?

Reply:
These are not declarations based on the user.
So the `using MyNamespace::g;` line does not use and does not declare the `g` function.

Reply:
The name *Resolution* seems quite arbitrary, and does not refer in any way that we are handling using statements with them, yet they are storing the location for the using statements. Maybe `UsingLoc` ?

Reply:
I renamed it to UsingLoc.